src/Client/TrapHandle/Dumper.php

```diff
@@ -22,66 +21,69 @@
 
 /**
  * @internal
+ *
  * @psalm-internal Buggregator\Trap\Client
  */
 final class Dumper
 {
-    /** @var null|Closure(mixed, string|null, int): mixed */
-    private static ?Closure $handler = null;
+    /** @var \Closure(mixed, string|null, int): mixed|null */
```

Comment:
I'm not sure that equals to ?Closure
It looks like a Closure that returns mixed|null

Need to test with IDE and Psalm